File: app/src/main/java/com/example/halilarm/Fragment4.java
package com.example.halilarm;

import android.os.Bundle;
import android.support.annotation.NonNull;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;

public class Fragment4 extends Fragment {

    @Nullable
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        ViewGroup view = (ViewGroup) inflater.inflate(R.layout.fragment4, container, false);

        Button btbt = view.findViewById(R.id.btbt);
        final TextView tvtv = view.findViewById(R.id.tvtv);
        final EditText eded = view.findViewById(R.id.eet);

        btbt.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                String msg = eded.getText().toString();
                tvtv.append("\n" + msg);
            }
        });

        return view;
    }
}

File: app/src/main/java/com/example/halilarm/Fragment4.kt
package com.example.halilarm

import android.os.Bundle
import android.os.Handler
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.Button
import android.widget.EditText
import android.widget.TextView
import androidx.fragment.app.Fragment
import org.w3c.dom.Text
import java.io.DataInputStream
import java.io.DataOutputStream
import java.net.Socket

class Fragment4: Fragment() {

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        val view = inflater.inflate(R.layout.fragment4, container, false) as ViewGroup

        return view
    }

}